tests/store/crud/test_read.py

```diff
@@ -90,6 +90,19 @@ def test_get_analyses_for_case(analysis_store: MockStore, case_id: str):
     assert analyses
 
 
+def test_get_analyses_with_statuses(analysis_store: MockStore):
+    """Test getting analyses with status when it exists in the database."""
+    # GIVEN a store with an analysis
+
+    # WHEN accessing it by statuses
+    analyses: Optional[List[Analysis]] = analysis_store.get_analyses_with_statuses(
+        statuses=list(TrailblazerStatus.ongoing_statuses())
+    )
+
+    # THEN it should return analyses
+    assert analyses
```

Comment:
```suggestion
    assert analyses
```
We should add an assertion that the returned analysis has the status it was queried by

Reply:
Added